compiler/src/main/java/bullet/impl/ComponentProcessingStep.java

```diff
@@ -224,4 +272,22 @@ private boolean isVisibleFrom(Element target, PackageElement from) {
         throw new AssertionError();
     }
   }
+
+  private static int getNextPrime(int value) {
```

Comment:
The comments about it not being 100% reliable are due to the fact that it has to handle numbers that can be 1000s of bits long and for really large numbers it isn't 100% reliable. For the range of numbers used here (typically < 100) it will be 100% reliable.